Essentials/src/messages.properties

```diff
@@ -143,6 +143,8 @@ flying=flying
 flyMode=\u00a76Set fly mode\u00a7c {0} \u00a76for {1}\u00a76.
 foreverAlone=\u00a74You have nobody to whom you can reply.
 fullStack=\u00a74You already have a full stack.
+fullStackDefault=\u00a76Stack increment amount would overflow the max stack size,\u00a7c {0}\u00a76. Defaulting to max stack size.
+fullStackDefaultOversize=\u00a76Stack increment amount would overflow the oversized stack size,\u00a7c {0}\u00a76. Defaulting to oversized stack size.
```

Comment:
I feel like this is a bit technical for most users -- keep in mind that `/more` could be used by players, not just developers / server admins. They could probably be condensed to something like "Your stack has been set to its maximum size.", or maybe even absolutely nothing at all (because without arguments this is the normal operation of the command).